### ISSUE TITLE
Display subset of dimensions

### ISSUE THREAD

**Issue by studywolf** [nengo/nengo_gui#734]
Not sure if this is already an issue, but quick search couldn't find it. 

It would be nice to be able to choose to display different subsets of the data in figures as the system is simulating. In larger ones i'll have 5-10 dimensions being plotted and trying to track anything specific is a pain. I guess I could send only that dimensions to a node and plot from there, but it seems like a thing that would be useful to be able to do just as a graph option. I thin it usedddd to be maybe?

**Comment by tcstewar** [comment]
Definitely.  We used to have that in nengo 1.4, and it's been mentioned a few times for the SPA pointer plots too.
